components/map/MapZoneBottomSheetAttachment.tsx

```diff
@@ -0,0 +1,64 @@
+import * as Location from 'expo-location'
+import { Link } from 'expo-router'
+import { useCallback } from 'react'
+import { View } from 'react-native'
+
+import { MapRef } from '@/components/map/Map'
+import BottomSheetTopAttachment, {
+  BottomSheetTopAttachmentProps,
+} from '@/components/shared/BottomSheetTopAttachment'
+import FlexRow from '@/components/shared/FlexRow'
+import Icon from '@/components/shared/Icon'
+import IconButton from '@/components/shared/IconButton'
+import PressableStyled from '@/components/shared/PressableStyled'
+import Typography from '@/components/shared/Typography'
+import { useLocationPermission } from '@/modules/map/hooks/useLocationPermission'
+
+type Props = Omit<BottomSheetTopAttachmentProps, 'children'> & {
+  setFlyToCenter?: MapRef['setFlyToCenter']
+}
+
+const MapZoneBottomSheetAttachment = ({ setFlyToCenter, ...restProps }: Props) => {
+  const { permissionStatus } = useLocationPermission()
+  const onLocationPress = useCallback(async () => {
+    if (permissionStatus === Location.PermissionStatus.GRANTED) {
+      let location = await Location.getLastKnownPositionAsync()
+      if (!location) {
+        location = await Location.getCurrentPositionAsync()
+      }
+      setFlyToCenter?.([location.coords.longitude, location.coords.latitude])
+    }
+  }, [setFlyToCenter, permissionStatus])
+
+  return (
+    <BottomSheetTopAttachment {...restProps}>
+      <FlexRow cn="flex-1 p-[10px] pt-0 items-end">
```

Comment:
`p-[10px] pt-0` --> `px-2.5 pb-2.5`

I hadn some issues with arbitrary values that they didn't scale to `dp` units. I think it's safer to use `-2.5` that we have defined in tailwind config.

Reply:
It looked like that `p-[10px]` was different than `p-2.5` when I used it but this was probably caused by the hot reload and Map being present